<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFVideoPlayerClass</class>
 <widget class="QMainWindow" name="FFVideoPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FFVideoPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVolume">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>455</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal 
{
        height: 3px;
        background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(124, 124, 124), stop: 1.0 rgb(72, 71, 71));
}

QSlider::handle:horizontal 
{
        width: 8px;
        background: rgb(0, 255, 255);
        margin: -3px 0px -3px 0px;
        border-radius: 4px;
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Playtime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>455</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    color: white;
	background-color: rgb(53,53,53);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlayVideo">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>455</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFullScreen">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>455</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MySlider" name="timeSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>800</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
        height: 3px;
        background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(124, 124, 124), stop: 1.0 rgb(72, 71, 71));
}
QSlider::handle:horizontal {
        width: 8px;
        background: rgb(255, 255, 255);
        margin: -3px 0px -3px 0px;
}
</string>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetVolume">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>455</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCutImage">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>455</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_OpenLocalFiles"/>
    <addaction name="action_OpenNetStream"/>
    <addaction name="action_OpenCamera"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_GetSourceCode"/>
   </widget>
   <widget class="QMenu" name="menu_Set">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Set"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_OpenLocalFiles">
   <property name="text">
    <string>打开本地视频</string>
   </property>
  </action>
  <action name="action_OpenNetStream">
   <property name="text">
    <string>打开网络流</string>
   </property>
  </action>
  <action name="action_OpenCamera">
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>关于本软件</string>
   </property>
  </action>
  <action name="action_GetSourceCode">
   <property name="text">
    <string>获取源代码</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>MySlider</class>
   <extends>QSlider</extends>
   <header>MySlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="FFVideoPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
